mvlearn/preprocessing/preprocess.py

```diff
@@ -0,0 +1,158 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Authors: Alexander Chang, Gavin Mischler
+
+import numpy as np
+from sklearn import model_selection
+from itertools import chain
+from ..utils.utils import check_Xs_y, check_Xs
+
+
+def train_test_split(Xs, y=None, test_size=None, train_size=None,
+                     random_state=None, shuffle=True, stratify=None):
+    r'''
+    Splits multi-view data into random train and test subsets. This
+    utility wraps the train_test_split function from
+    sklearn.model_selection for ease of use.
+
+    Parameters
+    ----------
+    Xs : list of array-likes or numpy.ndarray
+             - Xs length: n_views
+             - Xs[i] shape: (n_samples, n_features_i)
+            The input data.
+
+    y : array, shape (n_samples,), default=None
+        The target values of the data.
+
+    test_size : float or int, default=None
+        If float, should be between 0.0 and 1.0 and represent the
+        proportion of the dataset to include in the test split. If
+        int, represents the absolute number of test samples. If None,
+        the value is set to the complement of the train size. If
+        train_size is also None, it will be set to 0.25.
+
+    train_size: float or int, default=None
+        If float, should be between 0.0 and 1.0 and represent the
+        proportion of the dataset to include in the train split. If
+        int, represents the absolute number of train samples. If None,
+        the value is automatically set to the complement of the test
+        size.
+
+    random_state: int or RandomState instance, default=None
+        Controls the shuffling applied to the data before applying the
+        split. Pass an int for reproducible output across multiple
+        function calls.
+
+    shuffle: bool, default=True
+        Whether or not to shuffle the data before splitting. If
+        shuffle=False then stratify must be None.
+
+    stratify: array-like, default=None
+        If not None, data is split in a stratified fashion, using
+        this as the class labels.
+
+    Returns
+    -------
+    splitting : list, length=2*n_views or length=2*n_views+2
+        List containing the training and testing datasets, and the
+        training and testing targets if y was provided.
+        - Xs_train : list of array-likes, training data.
+        - Xs_test : list of array-likes, testing data.
+        - y_train : numpy.ndarray, training targets, only if y provided
+        - y_test : numpy.ndarray, testing targets, only if y provided
+
+    Examples
+    --------
+    >>> from mvlearn.preprocessing import train_test_split
+    >>> import numpy as np
+    >>> RANDOM_STATE=10
+    >>> np.random.seed(RANDOM_STATE)
```

Comment:
don't fix global state but use np.random.RandomState

Reply:
not sure why this was even in the example